packages/css-data/src/property-parsers/linear-gradient.test.ts

```diff
@@ -0,0 +1,277 @@
+            alpha: 1,
+            b: 0,
+            g: 255,
+            r: 255,
+            type: "rgb",
+          },
+          hint: undefined,
+          position: undefined,
+        },
+      ],
+    });
+  });
+
+  test("parses linear-gradient with angle and color-stops", () => {
+    expect(
+      parseLinearGradient("linear-gradient(135deg, orange 60% 20%, 40%, cyan)")
+    ).toEqual({
+      angle: {
+        type: "unit",
+        unit: "deg",
+        value: 135,
+      },
+      sideOrCorner: undefined,
+      stops: [
+        {
+          color: {
+            alpha: 1,
+            b: 0,
+            g: 165,
+            r: 255,
+            type: "rgb",
+          },
+          hint: {
+            type: "unit",
+            unit: "%",
+            value: 20,
+          },
+          position: {
+            type: "unit",
+            unit: "%",
+            value: 60,
+          },
+        },
+        {
+          hint: {
+            type: "unit",
+            unit: "%",
+            value: 40,
+          },
+        },
+        {
+          color: {
+            alpha: 1,
+            b: 255,
+            g: 255,
+            r: 0,
+            type: "rgb",
+          },
+          hint: undefined,
+          position: undefined,
+        },
+      ],
+    });
+  });
+
+  test("parses linear-gradient with side-or-corer and multiple colors without stops", () => {
+    expect(
+      parseLinearGradient(
+        "linear-gradient(to top right, orange, yellow, blue, green)"
+      )
+    ).toEqual({
+      angle: undefined,
+      sideOrCorner: {
+        type: "keyword",
+        value: "to top right",
+      },
+      stops: [
+        {
+          color: {
+            alpha: 1,
+            b: 0,
+            g: 165,
+            r: 255,
+            type: "rgb",
+          },
+          hint: undefined,
+          position: undefined,
+        },
+        {
+          color: {
+            alpha: 1,
+            b: 0,
+            g: 255,
+            r: 255,
+            type: "rgb",
+          },
+          hint: undefined,
+          position: undefined,
+        },
+        {
+          color: {
+            alpha: 1,
+            b: 255,
+            g: 0,
+            r: 0,
+            type: "rgb",
+          },
+          hint: undefined,
+          position: undefined,
+        },
+        {
+          color: {
+            alpha: 1,
+            b: 0,
+            g: 128,
+            r: 0,
+            type: "rgb",
+          },
+          hint: undefined,
+          position: undefined,
+        },
+      ],
+    });
+  });
+
+  test("parses linear-gradient with multiple angles and color-stops", () => {
+    expect(
+      parseLinearGradient(
+        "linear-gradient(to right, red 20%, orange 20% 40%, yellow 40% 60%, green 60% 80%, blue 80% )"
+      )
+    ).toEqual({
+      angle: undefined,
+      sideOrCorner: {
+        type: "keyword",
+        value: "to right",
+      },
+      stops: [
```

Comment:
I usually collapse long data like this into single line and let prettier do the magic